driver/src/contracts/stablex_contract.rs

```diff
@@ -4,17 +4,19 @@
 use crate::{
     contracts,
     models::{ExecutedOrder, Solution},
-    util::FutureWaitExt,
 };
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, Error, Result};
 use ethcontract::{
     contract::Event,
     errors::{ExecutionError, MethodError},
+    prelude::Web3,
     transaction::{confirm::ConfirmParams, GasPrice, ResolveCondition},
-    web3::{futures::Future as _, types::Block},
-    Address, BlockNumber, PrivateKey, U256,
+    transport::DynTransport,
+    web3::types::Block,
+    Address, BlockNumber, PrivateKey, H256, U256,
```

Comment:
I think I like this new way of importing more and more... it makes it easier to see what comes from what crate at a glance.